.github/workflows/test.yml

```
@@ -236,14 +236,16 @@ jobs:
         env:
           SCCACHE_BUCKET: turborepo-sccache
           SCCACHE_REGION: us-east-2
-          RUSTC_WRAPPER: "sccache"
+          # Only use sccache if we're in the Vercel repo.
+          RUSTC_WRAPPER: ${{ !github.event.pull_request.head.repo.fork && "sccache" || "" }}
```

GH Actions apparently doesn't like double quotes:
> Wrapping with double quotes (") will throw an error.


[source](https://docs.github.com/en/actions/learn-github-actions/expressions#literals)
```suggestion
          RUSTC_WRAPPER: ${{ !github.event.pull_request.head.repo.fork && 'sccache' || '' }}
```